app/src/main/java/org/hackillinois/android/view/MainActivity.kt

```diff
@@ -79,9 +79,9 @@ class MainActivity : AppCompatActivity() {
 
                 val newFragment = when (view) {
                     bottomAppBar.homeButton -> HomeFragment()
-                    bottomAppBar.todayButton -> ScheduleFragment()
-                    bottomAppBar.mapButton -> MapsFragment()
-                    bottomAppBar.groupButton -> MentorFragment()
+                    bottomAppBar.scheduleButton -> ScheduleFragment()
+                    bottomAppBar.mapsButton -> MapsFragment()
+                    bottomAppBar.projectsButton -> MentorFragment()
```

Comment:
You might want to rename MentorFragment to ProjectFragment for consistency

Reply:
Good call. I'll let @RaghavSaini01 do this in his PR, since he's handling the MentorFragment.